app/views/layouts/_head_tag_content.html.erb

```diff
@@ -25,8 +25,13 @@ signed in %>
 
 <%= render 'shared/appearance_styles' %>
 
-<!-- Google Analytics -->
-<%= render partial: '/ga', formats: [:html] %>
+<% unless Hyrax.config.analytics == 'false' %>
+  <% if Hyrax.config.analytics_provider == 'google' %>
```

Comment:
it might be nice to helperize this:

```ruby
def hyrax_analytics_partial(provider: Hyrax.config.analytics_provider)
  case provider
  when 'google'
    'shared/ga'
  when 'matomo'
    'shared/matomo'
  end
end
```

then the view code becomes

```ruby
<% if Hyrax.config.analytics %>
  <%= render partial: hyrax_analytics_partial, formats: [:html] %>
<% end %>
```